services/wallet/saved_addresses.go

```diff
@@ -15,16 +15,18 @@ type savedAddressMeta struct {
 
 type SavedAddress struct {
 	Address common.Address `json:"address"`
-	// TODO: Add Emoji and Networks
-	// Emoji    string         `json:"emoji"`
+	// TODO: Add Emoji
+	// Emoji	string		   `json:"emoji"`
 	Name      string `json:"name"`
 	Favourite bool   `json:"favourite"`
-	ChainID   uint64 `json:"chainId"`
+	Networks  string `json:"chains"` // used with address only, not with ENSName
```

Comment:
nit, why naming this Networks, instead of ChainIDs, this is what we use in wallet?

Reply:
yes for: ChainShortNames